PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -21,15 +21,24 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return (1.0 * numberCorrectAnswers / numberOfQuestions * 100) + "%";
     }
 
     public static bool AskQuestion(Question question)
     {
         DisplayQuestion(question);
 
         string userGuess = GetGuessFromUser();
-        return DisplayResult(userGuess, question);
+        if(userGuess == "1" || userGuess == "2" || userGuess == "3")
```

Comment:
```suggestion
        if (int.TryParse(userGuess, out int guess) && 1 <= guess && guess <= 3)
```
Another way to accomplish this if you wanted to use int comparison instead of string comparisons